packages/react-components/src/components/MessageThread.tsx

```diff
@@ -152,6 +152,8 @@ export interface MessageThreadStyles extends BaseCustomStyles {
   chatItemMessageContainer?: ComponentSlotStyle;
   /** Styles for my chat message container. */
   myChatMessageContainer?: ComponentSlotStyle;
+  /** Styles for my chat message container in case of failure.  */
+  failedMyChatMessageContainer?: ComponentSlotStyle;
```

Comment:
This changes the public API for this component, can you run `rush build:all-flavors` to update the api.md files

Reply:
I ran the `rush build:all-flavors` command and pushed the api.md files